apps/src/storage/getColumnDropdown.js

```diff
@@ -38,11 +37,14 @@ export function getTables() {
   };
 }
 
+function getTableNameFromColumnSocket(socket, editor) {
```

Comment:
2 nits here:
- we can remove the `editor` param since we aren't using it
- add a comment indicating why we have to strip the first/last characters